webxrlayers-1.bs

```diff
@@ -732,13 +729,16 @@ The {{XREquirectLayerInit}} dictionary represents a set of configurable values t
 dictionary XREquirectLayerInit : XRLayerInit {
   XRRigidTransform? transform;
   float radius = 10.0;
-  float scaleX = 1.0;
-  float scaleY = 1.0;
-  float biasX = 0;
-  float biasY = 0;
+  float centralHorizontalAngle = 6.28318;
+  float upperVerticalAngle = 3.14159;
+  float lowerVerticalAngle = 3.14159;
 };
 </pre>
 
+The default value of {{XREquirectLayerInit/centralHorizontalAngle}} is π * 2.
+The default value of {{XREquirectLayerInit/upperVerticalAngle}} is π.
```

Comment:
```suggestion
The default value of {{XREquirectLayerInit/upperVerticalAngle}} is π/2.
```

Reply:
your suggestion is marked as outdated. Maybe you can try again?